packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -24,6 +24,26 @@ const cli = {
         describe: 'A list of themes for the test run.',
         default: ['terra-default-theme'],
       },
+      hostname: {
+        type: 'string',
+        describe: 'Automation driver host address.',
+      },
+      port: {
+        type: 'number',
+        describe: 'Automation driver port.',
+      },
+      baseUrl: {
+        type: 'string',
+        describe: 'The base url.',
+      },
+      suite: {
```

Comment:
How does an example of a suite look like? Is it just an array of the wdio test file names?

Reply:
Suites get defined in the config, so it would be an array of suite names:

https://webdriver.io/docs/organizingsuites.html#grouping-test-specs